packages/shared/src/archive-storybook/index.ts

```diff
@@ -28,8 +28,13 @@ export function buildArchiveStorybook(
 }
 
 function binPath() {
-  // eslint-disable-next-line global-require
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
   const packageJson = require('storybook/package.json');
+  const parsedVersion = parseInt(packageJson.version[0], 10);
+  const storybookPackageJsonHasBinObject = parsedVersion >= 9;
```

Comment:
This is super-hacky. Storybook's `package.json` in v9 has a string for `bin`, whereas previously it had an object with a `storybook` key on it.

@yannbf do you see a better way of autodetecting the bin path for the installed Storybook dependency?